conf/modules.config

```diff
@@ -1207,6 +1219,7 @@ process{
     // MULTIQC
     withName:'MULTIQC' {
         errorStrategy = {task.exitStatus == 143 ? 'retry' : 'ignore'}
-        ext.args = { params.multiqc_config ? "--config $multiqc_custom_config" : "" }
+        //ext.args = { params.multiqc_config ? "--config $multiqc_custom_config" : "" }
```

Comment:
Can this be commented back in? Or removed in not needed?

Reply:
ah good spot! this should be obsolete due to the new multiqc module